meson.build

```diff
@@ -88,12 +88,11 @@ feature_map = {
   'insecure-tftp-update'            : '-DBMCWEB_INSECURE_ENABLE_REDFISH_FW_TFTP_UPDATE',
   #'vm-nbdproxy'                     : '-DBMCWEB_ENABLE_VM_NBDPROXY',
   'vm-websocket'                    : '-DBMCWEB_ENABLE_VM_WEBSOCKET',
-  'ibm-lamp-test'                   : '-DBMCWEB_ENABLE_IBM_LAMP_TEST',
+  'ibm-led-ext'                     : '-DBMCWEB_ENABLE_IBM_LED_EXT',
```

Comment:
nitpick: I don't see too many of these abbreviated, does ibm-led-extensions or ibm-additional-leds 

Need a change here https://github.ibm.com/openbmc/openbmc/blob/1020-ghe/meta-ibm/recipes-phosphor/interfaces/bmcweb_%25.bbappend#L17 before this can go

Reply:
Done